node-express/database/pgsql.js

```diff
@@ -0,0 +1,32 @@
+const pgp = require('pg-promise')();
+let _pg
+
+/**
+ *  Connects to the PostgreSQL database and tests the connection is valid
+ *  Visit http://vitaly-t.github.io/pg-promise/ for docs
+ *  @return {Promise} returns Promise which resolves the PostgreSQL connection Object
+ *  */
+function connectToDB() {
+    return new Promise((resolve) => {
+        _pg = pgp(`postgres://birlcvav:19QZsLOLiARafPCZOatrZobWlRYF5XGL@qdjjtnkv.db.elephantsql.com:5432/birlcvav`);
```

Comment:
Get all the database configuration details from a central config file. Also, Do not expose your personal database configurations.